knex/10MSeed/seed.js

```diff
@@ -0,0 +1,91 @@
+var performance = require('perf_hooks').performance;
+
+const maxNumPictures = 8;
+var LinkedList = function() {
+  this.head = null;
+  this.tail = null;
+}
+LinkedList.prototype.addToTail = function(value) {
+  var node = this.Node(value);
+  if (this.head === null) {
+    this.head = node;
+  }
+  if (this.tail) {
+    this.tail.next = node;
+  }
+  this.tail = node;
+}
+
+LinkedList.prototype.Node = function(value) {
+  var node = {};
+  node.value = value;
+  node.next = null;
+  return node;
+};
+
+exports.seed = async function(knex) {
+  var t0 = performance.now();
+  var dataSize = 10000000;
+  var photoList = new LinkedList();
+  var houseList = new LinkedList();
+  let i = 0;
+  await knex('houses').del();
+  await knex('photos').del();
+  while(i < dataSize) {
```

Comment:
Does this while loop actually work? Have you tested it? Nice idea on using a LinkedList, Maybe you can add a remove from Head function to the LinkedList.